Apps/WebClient/src/NewClientApp/src/views/RegistrationView.vue

```diff
@@ -341,7 +340,8 @@ loadTermsOfService();
             </h4>
             <HtmlTextAreaComponent
                 :input="termsOfService?.content"
-                class="mb-3"
+                class="mb-3 overflow-y-auto border"
+                height="330px"
```

Comment:
This could be a situation where CSS would be encouraged.

Reply:
I know but I stuck to the hard no css rule... *cringe*